draft-ietf-tsvwg-multipath-dccp.mkd

```diff
@@ -1188,7 +1188,7 @@ feature number 11 is specified with an exemplary description as below:
 ~~~~
 {: #ref-MP_EXP title='Format of the MP_EXP Suboption'}
  
-Details as length and type of data remain to be defined according to the foreseen use by the experimenters.
+According to the foreseen use by the experimenters Data MUST be defined within the scope of the 8bit Length field.
```

Comment:
I think this needs to be improved in wording,

Reply:
Does https://github.com/markusa/ietf-multipath-dccp/pull/219/commits/0e9b4780e9c14c3bffcbcd65d249a0bf00d3aa98 address your concern?